src/transform/FooterMenu.js

```diff
@@ -0,0 +1,218 @@
+  Array.from(Polyfill.querySelectorAll(fragment,
+    '.hide-when-compact, .collapsed')).forEach(el => el.remove())
+  // Get the innerText
+  return Array.from(Polyfill.querySelectorAll(fragment,
+    'td[class$=mbox-text]')).map(el => el.innerText)
+}
+
+/**
+ * Extracts array of disambiguation page urls from document.
+ * @type {FooterMenuItemPayloadExtractor}
+ */
+const disambiguationTitlesArray = document => Array.from(
+  Polyfill.querySelectorAll(document,
+    'div#content_block_0 div.hatnote a[href]:not([href=""]):not([redlink="1"])'
+  )
+).map(el => el.href)
+
+/**
+ * Type representing kinds of menu items.
+ * @enum {MenuItemType}
+ */
+const MenuItemType = {
+  languages: 1,
+  lastEdited: 2,
+  pageIssues: 3,
+  disambiguation: 4,
+  coordinate: 5
+}
+
+/**
+ * Menu item model.
+ */
+class MenuItem {
+  /**
+   * MenuItem constructor.
+   * @param {!string} title
+   * @param {?string} subtitle
+   * @param {!MenuItemType} itemType
+   * @param {FooterMenuItemClickCallback} clickHandler
+   * @return {void}
+   */
+  constructor(title, subtitle, itemType, clickHandler) {
+    this.title = title
+    this.subtitle = subtitle
+    this.itemType = itemType
+    this.clickHandler = clickHandler
+    this.payload = []
+  }
+
+  /**
+   * Returns icon CSS class for this menu item based on its type.
+   * @return {!string}
+   */
+  iconClass() {
+    switch (this.itemType) {
+    case MenuItemType.languages:
+      return 'pagelib_footer_menu_icon_languages'
+    case MenuItemType.lastEdited:
+      return 'pagelib_footer_menu_icon_last_edited'
+    case MenuItemType.pageIssues:
+      return 'pagelib_footer_menu_icon_page_issues'
+    case MenuItemType.disambiguation:
+      return 'pagelib_footer_menu_icon_disambiguation'
+    case MenuItemType.coordinate:
+      return 'pagelib_footer_menu_icon_coordinate'
+    default:
+      return ''
+    }
+  }
+
+  /**
+   * Returns reference to function for extracting payload when this menu item is tapped.
+   * @return {FooterMenuItemPayloadExtractor}
+   */
+  payloadExtractor() {
+    switch (this.itemType) {
+    case MenuItemType.languages:
+      return undefined
+    case MenuItemType.lastEdited:
+      return undefined
+    case MenuItemType.pageIssues:
+      return pageIssuesStringsArray
+    case MenuItemType.disambiguation:
+      return disambiguationTitlesArray
+    case MenuItemType.coordinate:
+      return undefined
+    default:
+      return undefined
+    }
+  }
+}
+
+/**
+ * Makes document fragment for a menu item.
+ * @param {!MenuItem} menuItem
+ * @param {!Document} document
+ * @return {!DocumentFragment}
+ */
+const documentFragmentForMenuItem = (menuItem, document) => {
+  const item = document.createElement('div')
+  item.className = 'pagelib_footer_menu_item'
+
+  const containerAnchor = document.createElement('a')
+  containerAnchor.addEventListener('click', () => {
+    menuItem.clickHandler(menuItem.payload)
+  }, false)
```

Comment:
`useCapture` defaults to false.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Parameters